HW02_B_Mimic_starter.Rmd

```diff
@@ -42,26 +43,47 @@ data("diamonds")
 Using the diamonds dataset, make this graph:
 ```{r graph1 code, echo=FALSE}
 
-
+ggplot(diamonds, aes(cut, fill = clarity))+
+  geom_bar(position = "dodge")+
+  labs(title="My Diamond Collection",
+       subtitle="Boxplot representing the number of diamonds in my diamon collection by\ntype of cut quality and clarity of diamond")+
+  theme(plot.title = element_text(hjust=0.5, face="bold"))+
+  xlab("Dimond Cut")+
+  ylab("Number of Diamonds")+
+  annotate("text", x="Premium", y=4500, label="My Best Diamonds\nof course")+
+  annotate("rect", xmin="Premium", xmax="Ideal", ymin=0, ymax=5000, alpha=0.5)# I don't know how to move the shade and text more to the left/right, since x-aix is discrete. 
 ```
 
 ### Graph 2
 ```{r, echo = T, include = TRUE}
 data("iris")
+
 ```
 
 Using the iris dataset, make this graph:
 ```{r graph 2 code, echo=FALSE}
+library(plyr) 
 
-```
+neworder <- c("versicolor","setosa","virginica")
+ 
+iris2 <- arrange(transform(iris,
+             Species=factor(Species,levels=neworder)),Species)#I don't know if there is a simpler way to change order of the panels in facet_wrap
 
+ggplot(iris2,mapping = aes(x=Sepal.Length, y=Petal.Length, color= Species, shape= Species))+

```

Comment:
I think you will also need to change the shape and color of data points representing the first two values of Species. I changed them using the following code:
_scale_color_manual(values = c('#00ba38','#f87676', '#619bff')) +
scale_shape_manual(values=c(17,16,15))_
However, I wonder how to change the order of items in the legend box?

Reply:
That's a good catch! How do you find the hex color codes?

Reply:
> That's a good catch! How do you find the hex color codes?

I found them in a very dumb way... I pasted the graph in Microsoft Paint and used color picker to get the RGB codes of those colors. Eventually, I had the RGB codes converted to HEX codes using Google. I'm sure there is a better way to do this!